easybuild/easyconfigs/i/iccifort/iccifort-2019.4.227-GCC-8.3.0-2.32.eb

```diff
@@ -0,0 +1,42 @@
+# This is an easyconfig file for EasyBuild, see http://easybuilders.github.io/easybuild
+
+name = 'iccifort'
+version = '2019.4.227'
```

Comment:
@bartoldeman Correct subversion is `243`, not `227` (otherwise the sanity check fails).

How did this work for you?

Reply:
Actually I took the 227 from the intel-2019b proposal @8681.
This exact easyconfig did not work for me because our license server is not up to date... I tested with 2019.3.

Reply:
argh, Intel has changed the tarball in the mean time. I'll adjust the 227 to be 243.